Adyen/Utilities/BundleSPMExtension.swift

```diff
@@ -18,7 +18,7 @@ internal extension Bundle {
     static let core: Bundle = .init(for: FormView.self)
 
     /// The bundle in which the framework's resources are located.
-    static let coreInternalResources: Bundle = .module
+    public static let coreInternalResources: Bundle = .module
```

Comment:
We should make `internal extension Bundle { .. }`  as `@_spi(AdyenInternal)` ?

Reply:
We need this to be `@_spi(AdyenInternal) public` because it's used from within the `AdyenComponents` now

Reply:
Currently *Adyen/Utilities/BundleSPMExtension.swift* is not under `@_spi(AdyenInternal)`

Reply:
Ahh I see, will change to be consistent with `CoreBundleExtension`

Reply:
Fixed here: f19c9ffed324a42053c0c85a6dab07855e78b89a